contracts/dex_grpc/Cargo.toml

```diff
@@ -32,7 +32,7 @@ library = []
 
 [dependencies]
 neutron-sdk = { workspace = true }
-cosmwasm-std = { workspace = true, features = ["stargate"] }
+cosmwasm-std = { workspace = true }
```

Comment:
I think there should be `grpc` feature instead?

Reply:
Not really. It could be [cosmwasm_2_0](https://github.com/CosmWasm/cosmwasm/blob/main/packages/std/src/query/mod.rs#L69), but it seems redundant, because that feature is already [used](https://github.com/neutron-org/neutron-sdk/blob/4a43e06115fe2768d9fa54c996270eb51694de39/packages/neutron-std-derive/Cargo.toml#L21) inside `neutron-sdk`.